App/l10n/Resources/tr.lproj/BraveShared.strings

```diff
@@ -1879,6 +1882,9 @@
 /* Recent Search suggestions title when prompting to turn on suggestions */
 "RecentSearchSuggestionsTitle" = "Arama Önerileri";
 
+/* Setting title for option to automatically redirect a url to another url */
+"redditRedirectFooter" = "Reddit bağlantılarını otomatik olarak (old.reddit.com adresindeki) daha eski arayüze bağlar. Yeni Reddi arayüzünü kullanmak için, bunun yerinenew.reddit.com adresini kullanın.";
```

Comment:
There is a small problem with spacing after word `yerine`

Corrected version:

Reddit bağlantılarını otomatik olarak (old.reddit.com adresindeki) daha eski arayüze bağlar. Yeni Reddi arayüzünü kullanmak için, bunun yerine new.reddit.com adresini kullanın.